openstack/utils/templates/_linkerd.tpl

```diff
@@ -0,0 +1,12 @@
+{{- define "utils.linkerd.pod_and_service_annotation" }}
+{{- if and $.Values.global.linkerd_enabled $.Values.global.linkerd_requested }}
+linkerd.io/inject: enabled
```

Comment:
This is just one line...

Reply:
It's just one line in the chart using this e.g.
```
{{- include "utils.linkerd.pod_and_service_annotation" . | indent 8 }}
```